src/networks.js

```diff
@@ -90,18 +136,6 @@ const NETWORKS = {
   },
 };
 
-/**
- * @type {Record<string, UrlBuilder>}
- */
-const urlBuilderMap = Object.fromEntries(
-  Object.entries(NETWORKS)
-    .map((entry) => [
-      entry[0],
-      entry[1].url,
```

Comment:
Instead of removing this whole chunk, why don't you redefine `urlBuilderMap` to include both `url` or `onclick` property:

```js
const urlBuilderMap = Object.fromEntries(
  Object.entries(NETWORKS)
    .map((entry) => [
      entry[0],
      {
        url: entry[1].url,
        property: entry[1].property,
      },
    ]),
);
```

In `shareon.js` you can use both just by referencing either `urlBuilderMap[cls].url` or `urlBuilderMap[cls].onclick`

Reply:
Yes, I started out doing it like that and then thought "why not just use fullNetworkMap?"  Now you have told me why :-)

Reply:
I will work on fixing this.  Do you prefer a rebased PR, or additional commits on top of this one?

Reply:
Just add commits on top of this one :)